README.md

```diff
@@ -214,15 +208,7 @@ Required settings:
   rather than email
 * `JANKY_CHAT_DEFAULT_ROOM` should be the name of the room instead of the
   XMPP format, for example: `Engineers` instead of xxxx_xxxxxx.
-
```

Comment:
Sorry, I wasn't clear enough before, since this installation section below is also for HipChat, let's not remove it in this PR.